x-pack/test/alerting_api_integration/observability/threshold_rule/avg_pct_fired.ts

```diff
@@ -0,0 +1,182 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { cleanup, generate } from '@kbn/infra-forge';
+import { Aggregators, Comparator } from '@kbn/observability-plugin/common/threshold_rule/types';
+import { FIRED_ACTIONS_ID } from '@kbn/observability-plugin/server/lib/rules/threshold/threshold_executor';
+import expect from '@kbn/expect';
+import { OBSERVABILITY_THRESHOLD_RULE_TYPE_ID } from '@kbn/observability-plugin/common/constants';
+import { createIndexConnector, createRule } from '../helpers/alerting_api_helper';
+import { createDataView, deleteDataView } from '../helpers/data_view';
+import { waitForAlertInIndex, waitForRuleStatus } from '../helpers/alerting_wait_for_helpers';
+import { FtrProviderContext } from '../../common/ftr_provider_context';
+
+// eslint-disable-next-line import/no-default-export
+export default function ({ getService }: FtrProviderContext) {
+  const esClient = getService('es');
+  const supertest = getService('supertest');
+  const esDeleteAllIndices = getService('esDeleteAllIndices');
+  const logger = getService('log');
+
+  describe('Threshold rule - AVG - PCT - FIRED', () => {
+    const THRESHOLD_RULE_ALERT_INDEX = '.alerts-observability.threshold.alerts-default';
+    const ALERT_ACTION_INDEX = 'alert-action-threshold';
+    const DATA_VIEW_ID = 'data-view-id';
+    let infraDataIndex: string;
+    let actionId: string;
+    let ruleId: string;
+
+    before(async () => {
+      infraDataIndex = await generate({ esClient, lookback: 'now-15m', logger });
+      await createDataView({
+        supertest,
+        name: 'metrics-fake_hosts',
+        id: DATA_VIEW_ID,
+        title: 'metrics-fake_hosts',
+      });
+    });
+
+    after(async () => {
+      await supertest.delete(`/api/alerting/rule/${ruleId}`).set('kbn-xsrf', 'foo');
```

Comment:
We have a utility [`ObjectRemover`](https://github.com/elastic/kibana/blob/main/x-pack/test/alerting_api_integration/common/lib/object_remover.ts) which may simplify some of this sort of stuff - or not!  Example usage [here](https://github.com/elastic/kibana/blob/main/x-pack/test/alerting_api_integration/security_and_spaces/group2/tests/alerting/mustache_templates.ts).